treeherder/etl/bugzilla.py

```diff
@@ -45,15 +51,19 @@ def run(self):
                 # we currently don't support timezones in treeherder, so
                 # just ignore it when importing/updating the bug to avoid
                 # a ValueError
-                Bugscache.objects.update_or_create(
-                    id=bug['id'],
-                    defaults={
-                        'status': bug.get('status', ''),
-                        'resolution': bug.get('resolution', ''),
-                        'summary': bug.get('summary', ''),
-                        'crash_signature': bug.get('cf_crash_signature', ''),
-                        'keywords': ",".join(bug['keywords']),
-                        'os': bug.get('op_sys', ''),
-                        'modified': dateutil.parser.parse(
-                            bug['last_change_time'], ignoretz=True)
-                    })
+                try:
+                    Bugscache.objects.update_or_create(
+                        id=bug['id'],
+                        defaults={
+                            'status': bug.get('status', ''),
+                            'resolution': bug.get('resolution', ''),
+                            'summary': smart_text(
+                                bug.get('summary', '')[:max_summary_length]),
+                            'crash_signature': bug.get('cf_crash_signature', ''),
+                            'keywords': ",".join(bug['keywords']),
+                            'os': bug.get('op_sys', ''),
+                            'modified': dateutil.parser.parse(
+                                bug['last_change_time'], ignoretz=True)
+                        })
+                except Exception, e:
```

Comment:
This is Python <=2.5 syntax, Python 2.6 support (and recommend) `Exception as e` which is Python 3 compatible :-)

https://www.python.org/dev/peps/pep-3110/#compatibility